fixtures/empty-block.go

```diff
@@ -0,0 +1,33 @@
+// Test of empty-blocks.
+
+package fixtures
+
+func f(x int) bool { // MATCH /this block is empty, you can remove it/
+
+}
+
+func g(f func() bool) string {
+	{ // MATCH /this block is empty, you can remove it/
+	}
+
+	if ok := f(); ok { // MATCH /this block is empty, you can remove it/
+		// only a comment
```

Comment:
Java's Checkstyle warns on empty blocks (with/without nested comments), because this "problem" may be caused by devs inadvertently commenting out full blocks of code and forgetting to un-comment them. Anyway I can modify the rule to only warn on "really" empty blocks or accept a configuration flag to take comments into account or not (defaulting to `false`)

Reply:
Let's keep it the way it is. If anyone is unhappy, we can introduce a config property.